stix2viz/stix2viz/stix2viz.js

```diff
@@ -585,6 +588,42 @@ define(["nbextensions/stix2viz/d3"], function(d3) {
             return;
           }
           Object.keys(item).forEach(function(key, index) {
+
+            // Iterate through the body_multipart array and do relationship linking there
+            if (key === 'body_multipart') {
```

Comment:
This is kind of quick and dirty very much a copy-paste kind of solution. But like, creating an abstraction might not be warranted.